Resources/views/ContentType/view_content_type.html.twig

```diff
@@ -23,13 +24,66 @@
 
 {% block content %}
     <section class="ez-serverside-content">
-        <div class="ez-table-data is-flexible">
-            <div class="ez-table-data-container">
+        <div class="context-information">
+            <p class="left modified">
+            {{
+                "content_type.last_modified"|trans({
+                    "%date%": content_type.modificationDate|localizeddate(locale=app.request.locale),
+                    "%modifier%": modifier.login
+                })
+            }}
+            </p>
+        </div>
+
+        <div class="block">
+            <div class="left">
+                <div class="block">
```

Comment:
Ok, word for mouth, lets try. correct me if I'm wrong here @dpobel:

```diff
+        <div class="pure-g">
+            <div class="pure-u-1 pure-u-sm-1-2">
+                <div>
```

I'm assuming all `class ="blocks"` can skipped as this is not float based like legacy css was.